Source/Core/DolphinQt/Config/LogWidget.cpp

```diff
@@ -210,11 +211,14 @@ void LogWidget::Log(LogTypes::LOG_LEVELS level, const char* text)
     break;
   }
 
+  std::string str(text);
+  StringPopBackIf(&str, '\n');
   m_log_queue.push(
-      QStringLiteral("%1 <font color='%2'>%3</font>")
-          .arg(QString::fromStdString(std::string(text).substr(0, TIMESTAMP_LENGTH)),
+      QStringLiteral(
+          "<span style=\"white-space: pre\">%1<span style=\"color: %2\">%3</span></span>")
```

Comment:
Any reason this isn't just `"%1 <span style=\"color: %2; white-space: pre\">%3</span>"` ?
Also, lack of space between `%1` and `%3`, unless the `<span>` introduces some (which it shouldn't)

Reply:
Fixed.

Good catch. Indeed, I don't need the `white-space: pre` style for the timestamp and `span` doesn't introduce space.